config/prod.exs

```diff
@@ -33,7 +33,7 @@ config :tilex, Tilex.Repo,
   adapter: Ecto.Adapters.Postgres,
   url: System.get_env("DATABASE_URL"),
   pool_size: String.to_integer(System.get_env("POOL_SIZE") || "10"),
-  ssl: true
+  ssl: false
```

Comment:
I never managed to get this working since it expects to get a boolean and all of my efforts like `if System.get_env("POSTGRES_USE_SSL"), do: true, else: false` failed.

Reply:
If you're setting it to false because postgres is running on localhost, you could just use a socket instead of TCP:

```
config :tilex, Tilex.Repo,
  socket_dir: "/var/run/postgresql"

```